src/Main.tsx

```diff
@@ -1,25 +1,33 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Outlet } from 'react-router'
 import { VAR_ENCODE_TOKEN, VAR_USER_ID } from './utils/Route';
 import { ToastContainer, toast } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 import Navbar from './components/common/Navbar/Navbar';
 import Footer from './components/common/Footer/Footer';
 import AsideNavbar from './components/common/AsideNavbar/AsideNavbar';
```

Comment:
suggestion: you can import components from a single file, that will look much cleaner and readable

Reply:
i thought of it but they are not that frequently imported only here